spec/migration/migrator_spec.rb

```diff
@@ -0,0 +1,83 @@
+# frozen_string_literal: true
```

Comment:
Is it valuable to also test that the migration version stored in redis is changed?

Reply:
yeah good idea!